conf/modules.config

```diff
@@ -62,5 +70,12 @@ if (params.mode == 'colabfold') {
                 saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
             ]
         }
+        withName: 'DOWNLOAD_COLABFOLD_PARAMS' {
+            publishDir = [
+                path: {"${params.db}"},
```

Comment:
If we go for the subworkflow and modularization of the download we will need to change the name of this parameter, but anyway I find confusing that is called `db` I will change it to `colabfold_params`

Reply:
Yes, that's a useful suggestion as long as we separate the AF2 downloads subworkflow from the colabfold downloads subworkflow